### PR TITLE
Start button

### PR DESCRIPTION
**Pull request by Alianshark**
Add background(color:black) 
   with button 
   and discription of keyboards action 
at start of loading page. After activation button game starts.

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -25,6 +25,7 @@ function Step () {
   checkGroundRocketCollision();
   renderRocket();
   renderShip();
+  initKeybordMovement()
```
Comment (Gehinom) [inline app.js:28]: Move to startGame function

File: style.css [hunk 2]
```diff
@@ -32,3 +55,16 @@ p {
     min-height: 100%;
 }
 
+#div-progress-bar {
```
Comment (Gehinom) [inline style.css:58]: Delete this style sheet!

File: app.js (hunk 1 above)
Comment (DarthVanger) [reply app.js]: I fixed.

File: style.css (hunk 2 above)
Comment (DarthVanger) [reply style.css]: fixed

**Comment by DarthVanger** [comment]
Start game button merge

**Pull request merged**